app/router.js

```diff
@@ -37,6 +37,8 @@ Router.map(function() {
     this.route('ministers', { path: '/ministers' });
     this.route('overview', { path: '/overzicht' });
     this.route('users', { path:"/gebruikers"});
+    this.route('errors', { path: "/errors"});
+    this.route('logs', { path: "/logs"});
   });
```

Comment:
This adds 2 routes to the settings overview, but both of them relate to monitoring.
Would it be better to add just the 1 route with subroutes ?

Reply:
Fixed by removing the logs path as discussed with Niels